Source/ActorPool/Private/ActorPoolActor.cpp

```diff
@@ -0,0 +1,299 @@
+﻿#include "ActorPoolActor.h"
+
+#include "APPooledActorInterface.h"
+#include "ActorPoolSubSystem.h"
+
+#include <Engine/World.h>
+
+#if !( UE_BUILD_SHIPPING || UE_BUILD_TEST )
+static FAutoConsoleCommandWithWorld GActorPoolDestroyInstancesInPools(

```

Comment:
keep those in the subsystem CPP file (you access the subsystem in those functions)

Reply:
Done.